frontend/src/utils/offlineMessages.js

```diff
@@ -3,19 +3,27 @@
  */
 module.exports = {
   offline: {
-    de_DE: {
-      OFFLINE_MESSAGE_LINE1: 'Verbindung unterbrochen',
-      OFFLINE_MESSAGE_LINE2:
-        'Es treten Verbindungsprobleme auf. Bitte kontrolliere die Verbindung und lade die Seite neu.',
-      BAD_GATEWAY_LINE1: 'Instanz ist nicht verfügbar',
-      BAD_GATEWAY_LINE2: 'Es treten Verbindungsprobleme auf.',
+    de: {
+      badGateway: {
+        title: 'Instanz ist nicht verfügbar',
+        description: 'Es treten Verbindungsprobleme auf.',
+      },
+      noStatus: {
+        title: 'Verbindung unterbrochen',
+        description:
+          'Es treten Verbindungsprobleme auf. Bitte kontrolliere die Verbindung und lade die Seite neu.',
+      },
     },
-    en_US: {
-      OFFLINE_MESSAGE_LINE1: 'Connection lost.',
-      OFFLINE_MESSAGE_LINE2:
-        'There are some connection issues. Check connection and try to refresh the page',
-      BAD_GATEWAY_LINE1: 'Instance is not available',
-      BAD_GATEWAY_LINE2: 'There are some connection issues.',
+    en: {
```

Comment:
consider moving this part in some offline_en.json file.

Reply:
here is an example: https://github.com/metasfresh/metasfresh/blob/tasty_ham_uat/misc/services/mobile-webui/mobile-webui-frontend/src/apps/inventoryDisposal/index.js#L4

Reply:
![Screenshot 2021-12-08 at 12 26 58](https://user-images.githubusercontent.com/1708561/145192502-af755daf-c261-4d01-9d80-e992fcff6a98.png)
things are a bit more complex as there are some configs that do not exist in webUI setup. I would leave as it is in one single file and after that import thing is fixed we can do this. Agree ?